composer/callbacks/system_metrics_monitor.py

```diff
@@ -19,13 +20,56 @@
 
 __all__ = ['SystemMetricsMonitor']
 
+_GPU_METRICS = [
+   'gpu_percentage',
+   'memory_percentage',
+   'gpu_temperature_C',
+   'gpu_power_usage_W',
+]
+
 
 class SystemMetricsMonitor(Callback):
-    """Track system metrics."""
+    """Logs GPU and CPU metrics relevant to straggler detection across all ranks.
```

Comment:
```suggestion
    """Logs CPU/GPU system metrics.
```
nit: more concise. it can be used for things besides straggler detection and by default is across all ranks